frontend/src/main/pages/CoursesAddStaffPage.js

```diff
@@ -0,0 +1,50 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import AddCourseStaffForm from "main/components/Courses/AddCourseStaffForm";
+import { Navigate } from 'react-router-dom'
+import { useBackendMutation } from "main/utils/useBackend";
+import { toast } from "react-toastify";
+import { useParams } from "react-router-dom";
+
+export default function AddCourseStaffPage({storybook=false}) {
+    let { courseId } = useParams();
+    const objectToAxiosParams = (staff) => ({
+        url: "/api/courses/addStaff",
+        method: "POST",
+        params: {
+            courseId: staff.courseId,
```

Comment:
```suggestion
            courseId: courseId
```

You already have the course id from the `useParams` call.

Reply:
Fixed